packages/browser/src/__tests__/cookieless.test.ts

```diff
@@ -310,5 +310,32 @@ describe('cookieless', () => {
             expect(beforeSendMock.mock.calls[3][0].properties.$cookieless_mode).toEqual(true)
             expect(posthog.sessionRecording).toBeFalsy()
         })
+
+        it('should restart autocapture after opt_in_capturing in cookieless mode', async () => {
+            const { posthog } = await setup({
+                cookieless_mode: 'on_reject',
+            })
+
+            // Mock the autocapture startIfEnabled method
+            const mockStartIfEnabled = jest.fn()
+            const originalStartIfEnabled = posthog.autocapture?.startIfEnabled
+            if (posthog.autocapture) {
+                posthog.autocapture.startIfEnabled = mockStartIfEnabled
+            }
+
+            // Clear any previous startIfEnabled calls
+            mockStartIfEnabled.mockClear()
+
+            // Opt in
+            posthog.opt_in_capturing()
+
+            // Autocapture should be restarted immediately after opt-in
+            expect(mockStartIfEnabled).toHaveBeenCalledTimes(1)
```

Comment:
Hmm I don't think this is the right test

This is just testing an implementation detail.

Ideally we would test whether an autocapture event is sent when a simulated click happens